docs/ingest-from-cloud-foundry-runtime-f5a7c99.md

```diff
@@ -93,12 +93,44 @@ For more information about different contexts, tools, options, and best practice
     cf service-key <service-instance> <service-key>
     ```
 
-5.  Create a user provided service using the following the template filled with the values of the previous step and a user-provided-service-name of your choice:
+5.  (a) Create a user provided service using the following the template filled with the values of the previous step and a user-provided-service-name of your choice:
 
     ```
     cf cups <user-provided-service-name> -l https-batch://<ingest-username>:<ingest-password>@<ingest-endpoint>/cfsyslog?drain-type=all
     ```
 
+    (b) **Use mTLS if required:**
+
+    Use this if you need mTLS. You still embed the basic auth username/password into the drain URL, while the certificate material is supplied separately as parameters.
+
+    5.1 Prerequisites
```

Comment:
Actually it would be the first substep of b not ot 5. Same for below
```suggestion
    b.1 Prerequisites
```

Reply:
done.